### DISCUSSION TITLE
Dashy in bridge network can't access to a domaine with access control (on Synology)

### DISCUSSION THREAD

**Discussion by MilesTEG1** [Lissy93/dashy#335] [Help and Support]
Hello @Lissy93 
I'm using Dashy for several months now, and I'm happy to use it :D Thanks for your work 👍🏻 

I see recently that you added this option : 
![image](https://user-images.githubusercontent.com/29298312/141732218-7c8b7b6c-248e-4d32-9d16-796b392a89b1.png)

It works fine for an access with https://ip_service:port.

But it doesn't work for another type of access : https://router.my_domaine.tld   (so with 443 port) when I have restricted access for this domaine : 
- it's unavailable oustide my LAN
- it's available throught my VPN server on my router (Syno RT2600AC)
- it's available inside my LAN with AdGuard Home rewriting the URL to 192.168.x.y (witch is my NAS IP).
(I have set up a virtual IP address in order to let my NAS access to adguard Home, so normaly the containers too...)

On DSM, I set Access Control Profile for https://router.my_domaine.tld (and some other like https://dsm.my_domaine.tld).
I put my LANs IP CIDR : 192.168.2.0/24
I put my VPN IP CIDR : 192.168.2.10/24
I put the Dashy container IP : 172.28.0.2

But Dashy can't access to the domaine name... : 
![image](https://user-images.githubusercontent.com/29298312/141732091-d885313c-b8b1-4327-9b57-e3239d46c0f8.png)

I don't know why...
Because dashy can access to my vaultwarden vault (but this one is without access control).
So Dash can access to AdGuard Home, I see it in AdG journal request...

Do you have an idea on with IP I can try to let Dashy check the status of my services domaine name ?

Thanks in advance :)

**Comment by lissy93** [comment]
Hmm that's weird, are these services accesible from where your hosting Dashy? All the status checking happens within your network, so this should work.
It's likley an authorization, cors, or missing protocol error. What result do you get when you manually call the endpoint? Here's a [troubleshooting guide](https://github.com/Lissy93/dashy/blob/master/docs/status-indicators.md#troubleshooting-failing-status-checks) which should help.

**Reply by MilesTEG1** [reply]
> Hmm that's weird, are these services accesible from where your hosting Dashy? All the status checking happens within your network, so this should work. 
I'm hosting Dashy in Docker on my NAS Synology, and DSM is obviously on this NAS too :)
The router not, but it get the same error.
I can access both of them from my computer.

> It's likley an authorization, cors, or missing protocol error. What result do you get when you manually call the endpoint?
What do you mean ?

> Here's a [troubleshooting guide](https://github.com/Lissy93/dashy/blob/master/docs/status-indicators.md#troubleshooting-failing-status-checks) which should help.
I didn't understand everything, but I found a workaround with `Status Check URL` property by filling it with the LAN IP of my services and its port.

If I found another way, I'll let you know.
Thanks for your help :)
